docs/api/classifier/classification_report.rst

```diff
@@ -5,6 +5,13 @@ Classification Report
 
 The classification report visualizer displays the precision, recall, F1, and support scores for the model. In order to support easier interpretation and problem detection, the report integrates numerical scores with a color-coded heatmap. All heatmaps are in the range ``(0.0, 1.0)`` to facilitate easy comparison of classification models across different classification reports.
 
+=================   =================
+Visualizer           :class:`~yellowbrick.classifier.ClassificationReport`
+Quick Method         :func:`~yellowbrick.classifier.classification_report`
+Models               Classification
+Workflow             Model evaluation
+=================   =================
```

Comment:
Thank you for adding this summary to the docs! It looked like the links weren't working so I doublechecked; looks like this should be instead:
```
=================   =================
Visualizer           :class:`~yellowbrick.classifier.classification_report.ClassificationReport`
Quick Method         :func:`~yellowbrick.classifier.classification_report.classification_report`
Models               Classification
Workflow             Model evaluation
=================   =================
```